app/build.gradle

```diff
@@ -70,6 +70,7 @@ dependencies {
     compile "com.android.support:cardview-v7:$rootProject.supportLibraryVersion"
     compile "com.google.android.gms:play-services-maps:$rootProject.playServicesVersion"
     compile "com.android.support:recyclerview-v7:$rootProject.supportLibraryVersion"
+    compile "com.android.support:gridlayout-v7:$rootProject.supportLibraryVersion"
```

Comment:
yeah do that, if it is not affecting the UI and please a Button to show and hide passcode